commitserver/commit/hydratorhelper.go

```diff
@@ -204,6 +204,17 @@ func writeManifests(root *os.Root, dirPath string, manifests []*apiclient.Hydrat
 		if err != nil {
 			return fmt.Errorf("failed to unmarshal manifest: %w", err)
 		}
+		// Remove Argo CD's internal tracking annotation before committing to Git.
```

Comment:
Instead of doing this in hydratorhelper.go immediately before commiting, can we do it in hydrator.go immediately after calling GetRepoObjs? I'd like to keep manifest-mutation logic out of the commit server.

Reply:
resolved.